pkg/api/alerting.go

```diff
@@ -353,10 +353,12 @@ func NotificationTest(c *models.ReqContext, dto dtos.NotificationTestCommand) Re
 
 //POST /api/alerts/:alertId/pause
 func PauseAlert(c *models.ReqContext, dto dtos.PauseAlertCommand) Response {
+	const Pause = "paused"
```

Comment:
we have const for this in models.AlertStatePaused

Reply:
Hi @torkelo ...I have modified the code to refer const defined in models.